modules/core/shared/src/main/scala/lucuma/core/syntax/IterableOps.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.syntax
+
+import scala.collection.{IterableOps => ScalaIterableOps}
+
+trait IterableOps:
+  extension [S[_], A](aa:  ScalaIterableOps[Option[A], S, S[Option[A]]])
+    /** Idiomatic way of reducing an `Iterable[Option[A]]` to `Iterable[A]` */
+    def unNone: S[A] =  aa.flatten
```

Comment:
Isn't this called `flattenOption` in `List`. Should use that name?

Reply:
For that matter is it maybe covered by `FilterFunctor`?

Reply:
Yup, it seems to be redundant.

I liked the `unNone` name: it's shorter, more specific and consistent with `fs2.Stream`'s, but it would introduce a 3rd way of achieving the same thing ( there's already `flatten` and `flattenOption`).